students/HABTAMU/session07/html_render.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python3
+
+"""
+A class-based system for rendering html.
+"""
+
+# This is the framework for the base class
+class Element(object):
+
+    tag = "html"
+    indent = "  "
+    # html_attributes = {"pid": "intro", "style": "text-align: center", "width":"400"}
+    html_attributes = dict()
```

Comment:
why a class attribute for this?

it'll get overwritten by the __init__ anyway.